source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,357 @@
 
-#. Power off the |CL| VM.
-
-   .. image:: ./figures/vbox/vbox-shutdown-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-08.png
+      :scale: 100%
       :alt: Powering off a VirtualBox VM
 
-#. Click *Settings* to configure the |CL| VM.
+      Figure 8: Powering off a VirtualBox VM
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+#. Click *Settings* to configure the |CL| VM.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
+#. From the *VM - Settings* window, navigate to the *Storage* pane from the
    left-hand side.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled  
-   *clear-<VERSION>-installer.iso* under the *Controller: IDE* from.
+#. From the middle *Storage Devices* column, click the blue CD disk labeled
+   *clear-<VERSION>-live-server.iso* under the *Controller: IDE* from.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Remove Disk from Virtual 
+#. From the right-hand *Attributes* column, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Remove Disk from Virtual
    Drive*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-unmount-ISO.png
-      :alt: Unmounting an ISO in VirtualBox VM Settings        
+   .. figure:: figures/vbox/virtualbox-cl-installer-09.png
+      :scale: 100%
+      :alt: Remove Disk from Virtual Drive
+
+      Figure 09: Remove Disk from Virtual Drive
 
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
+#. Click *OK* to exit the *VM Settings* menu and return to the main
    |VBM|.
 
+Install |VB| Linux Guest Additions
+==================================
 
+|CL| provides Linux Guest Additions drivers for full compatibility and an install script in the **kernel-lts** (Long Term Support) bundle by |CL|.
 
+#. From the |VBM| select the |CL| VM, and select :guilabel:`Start`.
 
-.. include:: virtualbox.rst
-   :start-after: vbox-start-vm-and-lga-begin:
-   :end-before: vbox-start-vm-and-lga-end:
+#. Log in and open a Terminal.
 
+#. Validate the installed kernel is **kernel-lts** by checking the output
+   of the :command:`uname -r` command. It should end in **.lts** or **.lts2018**.
 
+   .. code-block:: bash
+
+      uname -r
+      4.<VERSION>.lts
+
+   If the running kernel is not **lts**: install the LTS kernel manually,
+   update the bootloader, and check again:
+
+   .. code-block:: bash
+
+      sudo swupd bundle-add kernel-lts
+      clr-boot-manager set-kernel $(basename $(realpath /usr/lib/kernel/default-lts))
+      clr-boot-manager update
+      reboot
+
+#. Remove any kernel bundles that are not *kernel-lts* or *kernel-install*
+   to simplify and avoid conflicts:
+
+   .. code-block:: bash
+
+      swupd bundle-list | grep kernel
+      swupd bundle-remove <NON-LTS-KERNEL>
```

Comment:
Prefix with sudos here to be consistent with the changes below implying non-root user.